sections/main-product.liquid

```diff
@@ -359,6 +143,82 @@
                   {{ product.description }}
                 </div>
               {%- endif -%}
+              {%- when 'carousel' -%}
+                <cross-sell-carousel>
+                   <div class="glide" data-glide-cross-sell>
+                     <div class="glide__track" data-glide-el="track">
+                       <ul class="glide__slides">
+                         {% for product in collections[block.settings.collection].products %}
+                           <li class="glide__slide">
+                               <div class="card card--cross-sell">
+                               <img src="{{ product.featured_image | img_url: '134x134' }}"  alt="{{ product.title }}"/>
+                               <p class="cross-sell-product-title">{{ product.title }}</p>
+                               <div {{ block.shopify_attributes }}>
+                               {%- if product != blank -%}
+                               <product-form class="product-form">
+                                 <div class="product-form__error-message-wrapper" role="alert" hidden>
+                                    {% render 'svg-product-form-error' %}
+                                     <span class="product-form__error-message"></span>
+                                     </div>
+                                     {%- form 'product', product, id: product_form_id, class: 'form', novalidate: 'novalidate', data-type: 'add-to-cart-form' -%}
```

Comment:
Where is this being declared, if its not needed, can this be removed?

Reply:
Is this question about line 163?